src/widgets/splits/Split.cpp

```diff
@@ -807,7 +808,60 @@ void Split::addShortcuts()
              }
              return {};
          }},
-    };
+        {"setHighlightSounds",
+         [this](std::vector<QString> arguments) -> QString {
+             if (!this->getChannel()->isTwitchChannel())
+             {
+                 return "Cannot set highlight sounds in a non-Twitch "
+                        "channel.";
+             }
+
+             auto mode = 2;
+             // 0 is off
+             // 1 is on
+             // 2 is toggle
+             if (!arguments.empty())
+             {
+                 auto arg = arguments.at(0);
+                 if (arg == "off")
+                 {
+                     mode = 0;
+                 }
+                 else if (arg == "on")
+                 {
+                     mode = 1;
+                 }
+                 else
+                 {
+                     mode = 2;
+                 }
```

Comment:
`mode` is already initialized to 2.

Reply:
Sure the implementation was copied here.